clar2wasm/src/wasm_generator.rs

```diff
@@ -1653,7 +1653,7 @@ impl WasmGenerator {
                     builder.call(
                         self.module
                             .funcs
-                            .by_name("stdlib.block_height")
+                            .by_name("block_height")
```

Comment:
Sames as above. `stdlib.` prefix is now part of function present on Clarity standard library.